### PR TITLE
chore: clean up SkyWalking docs

### PR DESCRIPTION
**Pull request by Superskyyy**
### What this PR does / why we need it:

This PR cleans up and enhances en/zh documentation of the SkyWalking Plugin .
- Fixes grammar/vocab error
- Bumps up the SkyWalking version mentioned

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/plugins/skywalking.md [hunk 1]
```diff
@@ -78,20 +79,20 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-You also can complete the above operation through the web interface, first add a route, then add skywalking plugin:
+You also can complete the above operation through the web interface, first add a route, then add SkyWalking plugin:
```
Comment (tokers) [inline docs/en/latest/plugins/skywalking.md:82]: You also can => You can also

Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md:82]: This following line exists widely in many APISIX docs - You also can complete the above operation through the web interface, should I change it for this one? Note that this also can, though sounds a bit odd, it is indeed a correct usage identical to can also.

FYI @tokers

File: docs/en/latest/plugins/skywalking.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: skywalking
+title: SkyWalking
```
Comment (tzssangglass) [inline docs/en/latest/plugins/skywalking.md:2]: No need for camel naming, keep it lowercase like the rest of the plugin documentation.

File: docs/en/latest/plugins/skywalking.md [hunk 3]
```diff
@@ -27,24 +27,25 @@ title: skywalking
 - [**Attributes**](#attributes)
 - [**How To Enable**](#how-to-enable)
 - [**Test Plugin**](#test-plugin)
-  - [**Run Skywalking Example**](#run-skywalking-example)
+  - [**Run SkyWalking Example**](#running-skywalking)
```
Comment (tzssangglass) [inline docs/en/latest/plugins/skywalking.md:30]:
```suggestion
- [**Running SkyWalking**](#running-skywalking)
```

Comment (tzssangglass) [reply docs/en/latest/plugins/skywalking.md:30]: text and links should keep same

File: docs/en/latest/plugins/skywalking.md [hunk 4]
```diff
@@ -78,20 +79,20 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-You also can complete the above operation through the web interface, first add a route, then add skywalking plugin:
+You also can complete the above operation through the web interface, first add a route, then add SkyWalking plugin:
 
 ![ ](../../../assets/images/plugin/skywalking-1.png)
 
-## How to set endpoint
+## How to Set Endpoint
```
Comment (tzssangglass) [inline docs/en/latest/plugins/skywalking.md:86]: why change `set endpoint` to `Set Endpoint`? This title looks strange.

File: docs/en/latest/plugins/skywalking.md [hunk 5]
```diff
@@ -206,7 +208,7 @@ plugins:
 
 And then reload APISIX.
 
-## Upstream services(Code With SpringBoot)
+## Upstream Services(Code With SpringBoot)
```
Comment (tzssangglass) [inline docs/en/latest/plugins/skywalking.md:211]: ditto, this title looks strange.

File: docs/en/latest/plugins/skywalking.md (hunk 4 above)
Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md:86]: The level two ## titles are not using a consistent capitalization, half of them are all caps while the others aren't, so I was trying to make them match. The same goes to the endpoint one below.

File: docs/en/latest/plugins/skywalking.md (hunk 3 above)
Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md:30]: My bad, missed this one.

File: docs/en/latest/plugins/skywalking.md (hunk 2 above)
Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md:2]: Zipkin is capitalized and I was trying to keep consistent, I will revert this then.

File: docs/en/latest/plugins/skywalking.md (hunk 4 above)
Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md:86]: @tzssangglass Would you kindly suggest that I simply revert the changes or make the other ones consistent with the `Cap lower` style?

File: docs/en/latest/plugins/skywalking.md (hunk 5 above)
Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md]: Reverted.

**Review by tzssangglass — APPROVED** [review]

**Comment by dmsolr** [comment]
Hi, @Superskyyy I saw the picture of SkyWalking topology which is using Nginx logo as APISIX's logo. 
It is out of date in SkyWalking 8.7.0. Could you update the picture?

**Comment by Superskyyy** [comment]
> Hi, @Superskyyy I saw the picture of SkyWalking topology which is using Nginx logo as APISIX's logo.
> It is out of date in SkyWalking 8.7.0. Could you update the picture?

Sure! Will update later.

**Comment by dmsolr** [comment]
> That is a screenshot from the original plugin contributor right? I will need to deploy APISIX to make a new ss, let's merge this one first and I will do the logo fix when I got time. What you think. @dmsolr

Yes. This is good to me to update the screenshot.

**Comment by Superskyyy** [comment]
Please review and merge this first, logo fix will be coming in another PR. @tokers Thanks.

File: docs/en/latest/plugins/skywalking.md [hunk 6]
```diff
@@ -26,25 +26,27 @@ title: skywalking
 - [**Name**](#name)
 - [**Attributes**](#attributes)
 - [**How To Enable**](#how-to-enable)
+- [**How to set endpoint**](#how-to-set-endpoint)
 - [**Test Plugin**](#test-plugin)
-  - [**Run Skywalking Example**](#run-skywalking-example)
+  - [**Running SkyWalking**](#running-skywalking)
```
Comment (spacewander) [inline docs/en/latest/plugins/skywalking.md:31]: Please use `Run` like other titles

File: docs/en/latest/plugins/skywalking.md [hunk 7]
```diff
@@ -26,25 +26,27 @@ title: skywalking
...
+[**SkyWalking**](https://github.com/apache/skywalking) uses its native Nginx LUA tracer to provide tracing, topology analysis, and metrics from service and URI perspective.
+
+The SkyWalking server can support both HTTP and gRPC protocols. Currently, the APISIX client only supports the HTTP protocol.
 
 ## Attributes
 
 | Name         | Type   | Requirement | Default  | Valid        | Description                                                          |
 | ------------ | ------ | ----------- | -------- | ------------ | -------------------------------------------------------------------- |
-| sample_ratio | number | required    | 1        | [0.00001, 1] | the ratio of sample                                                  |
+| sample_ratio | number | required    | 1        | [0.00001, 1] | The ratio of sampling                                               |
```
Comment (spacewander) [inline docs/en/latest/plugins/skywalking.md:45]: Please use lowercase for the first word like other cells

File: docs/en/latest/plugins/skywalking.md [hunk 8]
```diff
@@ -78,20 +80,20 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
...
 | ------------ | ------ | -------- | -------------------------------------------------------------------- |
-| service_name | string | "APISIX" | service name for skywalking reporter                                 |
-| service_instance_name | string |"APISIX Instance Name" | service instance name for skywalking reporter，  set it to `$hostname` to get local hostname directly.|
-| endpoint_addr | string | "http://127.0.0.1:12800" | the http endpoint of Skywalking, for example: http://127.0.0.1:12800 |
-| report_interval | integer | use the value in the skywalking client library | the report interval, in seconds |
+| service_name | string | "APISIX" | service name for SkyWalking reporter                                 |
+| service_instance_name | string |"APISIX Instance Name" | service instance name for SkyWalking reporter，  set it to `$hostname` to get local hostname directly.|
+| endpoint_addr | string | "http://127.0.0.1:12800" | the HTTP endpoint of SkyWalking, for example: http://127.0.0.1:12800 |
+| report_interval | integer | uses the value in the SkyWalking client library | the report interval, in seconds |
```
Comment (spacewander) [inline docs/en/latest/plugins/skywalking.md:96]:
```suggestion
| report_interval | integer | use the value in the SkyWalking client library | the report interval, in seconds |
```

File: docs/en/latest/plugins/skywalking.md (hunk 6 above)
Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md:31]: My bad, fixed

File: docs/en/latest/plugins/skywalking.md (hunk 7 above)
Comment (Superskyyy) [reply docs/en/latest/plugins/skywalking.md]: Ok, fixed

**Review by moonming — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Comment by Superskyyy** [comment]
Just a heads up SkyWalking 8.8.0 OAP is going to release very soon.

**Comment by moonming** [comment]
> Just a heads up SkyWalking 8.8.0 OAP is going to release very soon.

Is there anything need to modify for Apache APISIX?

**Comment by Superskyyy** [comment]
> > Just a heads up SkyWalking 8.8.0 OAP is going to release very soon.
> 
> 
> 
> Is there anything need to modify for Apache APISIX?

I'm not aware of that yet, I can help run a test when it's released.

**Comment by moonming** [comment]
> > > Just a heads up SkyWalking 8.8.0 OAP is going to release very soon.
> > 
> > 
> > Is there anything need to modify for Apache APISIX?
> 
> I'm not aware of that yet, I can help run a test when it's released.

Great！

**Pull request merged**